app/models/services/service_binding.rb

```diff
@@ -3,6 +3,7 @@
 module VCAP::CloudController
   class ServiceBinding < Sequel::Model
     include Serializer
+    include VCAP::CloudController::Presenters::Mixins::MetadataPresentationHelpers
```

Comment:
Already thinking this is not a great idea. 
Might move the `required_parameters` to the client since it only relates to OSBAPI interface

Reply:
Agreed. It has the potential to become a circular dependency, as the presenters depend on the models; do we don't really want the models to depend on the presenters...